randomImages/app.js

```diff
@@ -0,0 +1,18 @@
+const button  = document.querySelector('#randomize');
+const  addImg =function() {
+// console.log("test)
+const Widtht= document.querySelector('.width').value;
+const height =document.querySelector('.height').value;
+const newSrc= "https://source.unsplash.com/"+Widtht+"x"+height;
+
+console.log(newSrc);
+
```

Comment:
try to make your code run on functions to make your code reusable and easier to read